File: javascript/Flowcontrols/Looping/amstrong.js
var num=153;
let res=0;
let temp=num;
while(num>0){
    digit=num%10;
    res=res+digit**3;
    num=num/10;
}
if(num==res){
    console.log(num,"is an amstrong number");
}
else{
    console.log(num,"is not an amstrong number");
}
console.log(res);

File: flowcontrols/looping/amstrong.py
num=int(input("enter the num"))
res=0
while(num>0):
    digit=num%10
    res=res+digit**3
    num=num//10
if (num==res):
    print(num,"is an amstrong number")
else:
    print(num,"is not an amstrong number")
print(res)
    